runtime/drivers/druid/druid_test.go

```diff
@@ -86,13 +86,9 @@ func TestDruid(t *testing.T) {
 	container, err := testcontainers.GenericContainer(ctx, testcontainers.GenericContainerRequest{
 		Started: true,
 		ContainerRequest: testcontainers.ContainerRequest{
-			ExposedPorts: []string{"8081/tcp", "8082/tcp"},
 			WaitingFor:   wait.ForHTTP("/status/health").WithPort("8081"),
-			FromDockerfile: testcontainers.FromDockerfile{
-				Context:       ".",
-				Dockerfile:    "Dockerfile",
-				PrintBuildLog: true,
-			},
+			Image:        "druid-micro:latest",
```

Comment:
I think this will only work on your local because the image name is cached. It needs the full path of the image. Let's also pin to an exact version to prevent caching issues. So I think it will be `gcr.io/rilldata/druid-micro:25.0.0`?